Stripe/Checkout/STPCheckoutViewController.m

```diff
@@ -86,11 +86,11 @@ @interface STPCheckoutViewController () <STPCheckoutDelegate>
 @implementation STPCheckoutViewController
 
 - (instancetype)initWithNibName:(__unused NSString *)nibNameOrNil bundle:(__unused NSBundle *)nibBundleOrNil {
-    return [self initWithOptions:nil];
+    return [self initWithOptions:[[STPCheckoutOptions alloc] init]];
```

Comment:
I've changed the interface for `STPCheckoutViewController` just slightly to annotate its `checkoutOptions` as a `nonnull` property - in other words, that value should never be `nil`. So you should always create an `STPCheckoutViewController` with a valid instance of `STPCheckoutOptions`. While `initWithOptions:` is the designated initializer for `STPCheckoutViewController`, it might not always be initialized that way - for example, it might be loaded from a nib (in which case `initWithCoder:` would be called instead). In this case I think it's slightly preferable to load an empty `STPCheckoutOptions` (which will be preconfigured with your publishable key, if you've set it via `[Stripe setDefaultPublishableKey:]`, which we encourage) instead of a `nil` object. Either way, the proper thing to do in this case would be to explicitly set the `checkoutOptions` property before presenting an alternatively-instantiated `STPCheckoutViewController`.